cosmos/job/drm/drm_ge.py

```diff
@@ -195,14 +207,24 @@ def _is_corrupt(qacct_dict):
            ("before writing exit_status" not in qacct_dict.get('failed', ''))
 
 
-def _qacct_raw(task, timeout=600, quantum=15):
+def qacct(job_id, timeout=1200, quantum=15, logger=None, log_prefix=""):
```

Comment:
I changed the signature so that someone could import `from cosmos... import qacct` and get all of this functionality/error-handling without having to instantiate a Task object. Three guesses as to who _someone_ is.